src/components/new-request/search.vue

```diff
@@ -9,48 +9,62 @@
     <v-row class="mt-6">
       <!-- Request Action -->
       <v-col cols="12" md="6" class="py-0">
```

Comment:
The change below looks cleaner if you go to ⚙️ and then hide whitespace changes.